play-crate-io/crate.service

```diff
@@ -36,8 +36,8 @@ ExecStart=/bin/bash -c '\
               -Des.discovery.zen.minimum_master_nodes=2 \
               -Des.discovery.zen.ping.unicast.hosts=$HOSTS"'
 
-ExecStop=/bin/bash -c "docker stop `docker ps -l | gawk '{ match($0, /([a-z0-9]+)\s+crate:latest/, res); if(length(res[1]) > 1) print res[1]}'`"
-ExecStopPost=/bin/bash -c "docker rm `docker ps -l | gawk '{ match($0, /([a-z0-9]+)\s+crate:latest/, res); if(length(res[1]) > 1) print res[1]}'`"
+ExecStop=/bin/bash -c "docker stop `docker ps -l | gawk '{ match($0, /([a-z0-9]+)\s+crate\/crate:play/, res); if(length(res[1]) > 1) print res[1]}'`"
```

Comment:
'cause persistence: I would need to store the name somewhere and without knowing the clusters, a generated name will potentially have duplicates (or be very unhandy like UUIDs)